src/screens/Feed.js

```diff
@@ -17,11 +18,9 @@ class Feed extends Component {
   render() {
     return (
       <ScreenContainer screenTitle="Feed" >
-        <MainContainer isElevated backgroundColor={theme.COLOR_BLUE}>
-          <Title>Hola</Title>
-          <Subtitle>Amiga</Subtitle>
-        <TextBody>Classes!</TextBody>
-        </MainContainer>
+        <CustomMessage isAuthor={true} isSelectable={true}>
+          I LOVE PIE!
```

Comment:
remove

Reply:
The "I LOVE PIE!" message?  I do not have the message on my branch.

Reply:
Well it's part of the PR

Reply:
@nouj you might have forgotten to push your changes?